docker-compose.yml

```diff
@@ -10,7 +15,5 @@ services:
     volumes:
       - tofnd:/.tofnd
     environment:
-      # ATTENTION: use --unsafe only for testing
-      - UNSAFE=true
-      # available cmds: create (default), stored, update, import, export
-      - MNEMONIC_CMD=create 
+      # available cmds: auto, create, existing (default), import, export
```

Comment:
```suggestion
      # available cmds: auto (default), create, existing, import, export
```

Reply:
I see the source of confusion. `existing` is actually the default command here. Meaning that, if you don't explicitly define `- MNEMONIC_CMD=auto`, it will fallback to `existing`. So I think it's better to keep _existing (default)_.